.gitignore

```diff
@@ -154,3 +154,6 @@ automation/.bsp/*
 /config/
 /config_postgres/
 /env.properties
+
+## Shade
+dependency-reduced-pom.xml
```

Comment:
what does this do?

Reply:
Great question!  If you were going to give our shaded JAR/WAR to someone else, this pom conveys the minimal set of dependencies they would need to include to leverage our shaded archive..
There's a great discussion on SO about this: https://stackoverflow.com/questions/22904573/what-is-the-purpose-of-dependency-reduced-pom-xml-generated-by-the-shade-plugin

I see it as a nice to have kind of thing.  

There's been a request to the shade team to allow specifying the path of this file in the backlog for some time because the current implementation is broken. You can specify the root of the project, but anything else seems to cause breakage.

Reply:
I like having it in the ignore since it is generated by certain maven commands and can show up in your IDE as a new file that looks like it should be added.